test/integration/pack/test_pack.py

```diff
@@ -520,6 +526,36 @@ def prepare_tgz_test_cases(tt_cmd) -> list:
             ],
             "check_env": ["myapp", assert_single_app_env, assert_artifacts_env]
         },
+        {
+            "name": "Vshard app packing",
+            "bundle_src": "vshard_app",
+            "cmd": tt_cmd,
+            "pack_type": "tgz",
+            "args": ["--name", "test_app"],
+            "res_file": "test_app-0.1.0.0." + get_arch() + ".tar.gz",
+            "check_exist": [
+                os.path.join("bin", "tarantool"),
+                os.path.join("bin", "tt"),
+                os.path.join("instances.enabled", "test_app"),
+                os.path.join("tt.yaml"),
+                os.path.join("test_app/config.yaml"),
+                os.path.join("test_app/instances.yaml"),
+                os.path.join("test_app/router.lua"),
+                os.path.join("test_app/storage.lua"),
+            ],
+            "check_not_exist": [
+                os.path.join("test_app/test_app-scm-1.rockspec"),
+                os.path.join("test_app/tt.yaml"),
```

Comment:
Why we should use `join` here?
```suggestion
                "test_app/test_app-scm-1.rockspec",
                "test_app/tt.yaml",
```

Reply:
Removed.